casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -39,7 +38,7 @@ class RuntimeManager[F[_]: Concurrent: ToAbstractContext] private (
           result <- if (processedDeploy.status.isFailed) Seq.empty[Datum[ListParWithRandom]].pure[F]
                    else {
                      val r: F[Seq[Datum[ListParWithRandom]]] =
-                       ToAbstractContext[F].fromTask(runtime.space.getData(name))
+                       runtime.space.getData(name).map(_.toSeq)
```

Comment:
why is the `._toSeq` needed?

Reply:
afaik - types. we have a weird mix of immutable.Seq and mutable.Seq usage.

Reply:
`scala.collection.immutable.Seq` vs `scala.collection.Seq`

Reply:
I still don't get why it wasn't needed before :). Oh, is Task covariant? Yes, it is... Guess I have my answer :)